sdk/identity/azure_identity/src/refresh_token.rs

```diff
@@ -10,6 +10,7 @@ use azure_core::{json::from_json, Method};
 use serde::Deserialize;
 use std::fmt;
 use std::sync::Arc;
+use typespec_client_core::error::http_response_from_body;
```

Comment:
Do we want references to TypeSpec core showing up everywhere or should these be exposed from azure_core? This is a general design pattern

//cc @bterlson

Reply:
Probably re-export them from `azure_core`. I did that with all the others but was reluctant to ship these. That said, this is effectively an "internal" method so customers wouldn't be expected to use this in their code. It's for us only. In those cases, I don't really have a strong opinion on re-exporting or not from `azure_core`.

Reply:
Re-exported